zoneminder-latest.json

```
@@ -0,0 +1,80 @@
+                    }
+                },
+                "ports": {
+                    "80": {
+                        "description": "Insecure port for running ZoneMinder. You shouldn't open this externally (it's unsecure). Do not use 80 (it will interfere with RockStor)",
+                        "host_default": 8080,
+                        "protocol": "tcp",
+                        "label": "ZoneMinder Insecure Port"
+                    },
+                    "443": {
+                        "description": "Secure port for running ZoneMinder. This will use a self-signed SSL certificate for now.",
+                        "host_default": 8443,
+                        "protocol": "tcp",
+                        "label": "ZoneMinder Secure Port",
+                        "ui": true
+                    },
+                    "9000": {
+                        "description": "zmninja monitoring port",
+                        "host_default": 9000,
+                        "protocol": "tcp",
+                        "label": "ZMNinja Port"
+                    }
+                },
+                "volumes": {
+                    "/config": {
+                        "description": "Choose a Share for the config.",
+                        "label": "ZoneMinder Config",
+                        "min_size": "1073741824"
+                    },
+                    "/var/cache/zoneminder": {
+                        "description": "Choose a Share for the cache.",
+                        "label": "ZoneMinder Cache",
+                        "min_size": "1073741824"
+                    }
+                }
+            }
+        },
+        "description": "ZoneMinder: Free, open-source software to control IP, USB and Analog (CCTV) cameras (v1.32) - please note this runs as privileged in docker (to set shm to a higher amount)",
+        "more_info": "</h4>Tips and Setup Instructions:</h4></p>        </p>        This container includes mysql, no need for a separate mysql/mariadb container</p>            All settings and library files are stored outside of the container and they are preserved when this docker is updated or re-installed (change the variable /path/to/config in the run command to a location of your choice)</p>                    Important:</p>                    </p>                    The web gui will be available at https://serverip:port/zm</p>                    On first start, open zoneminder settings, go to the paths tab and enter the following for PATH_ZMS: /zm/cgi-bin/nph-zms</p>                        The default timezone for php is set as America/New_York if you would like to change it, edit the php.ini in the config folder. Here's a list of available timezone options: http://php.net/manual/en/timezones.php",
+        "ui": {
+            "https": true,
+            "slug": "zm"
+        },
+        "volume_add_support": true,
+        "version": "1.32",
+        "website": "https://github.com/dlandon/zoneminder"
```

Link to project itself rather than docker image:

```suggestion
        "website": "https://zoneminder.com"
```